tests/integration/models/release.spec.coffee

```diff
@@ -68,6 +76,26 @@ describe 'Release Model', ->
 						id: @currentRelease.id
 						belongs_to__application: __id: @application.id
 
+			it 'should get the requested release by commit', ->
+				balena.models.release.get(@currentRelease.commit)
+				.then (release) =>
+					m.chai.expect(release).to.deep.match
+						status: 'success',
+						source: 'cloud',
+						commit: 'new-release-commit',
+						id: @currentRelease.id
+						belongs_to__application: __id: @application.id
+
+			it 'should get the requested release by shorter commit', ->
+				balena.models.release.get(@currentRelease.commit.slice(0, 8))
```

Comment:
In the dashboard we show 7 characters of the commit hash, it's not that important for the tests but we should probably have a common method for getting a short hash for a commit (already did that in an upcoming staged releases PR for the dashboard). Not a blocker, I just thought I should mention it.